src/tree/utils/mergeAllOf.ts

```diff
@@ -28,7 +29,7 @@ function _mergeAllOf(schema: JSONSchema4, path: JsonPath, opts: WalkingOptions)
         schemaRefs = [$ref];
         allRefs.set(schema, schemaRefs);
       } else if (schemaRefs.includes($ref)) {
-        const safelyResolved = JSON.parse(safeStringify(opts.resolveRef(null, $ref)));
+        const safelyResolved = JSON.parse(stringify(opts.resolveRef(null, $ref)));
```

Comment:
is `stringify` less safe than `safeStringify`, I can't remember the difference

Reply:
They are equally safe in this case.
I mean, `safeStringify` may return undefined, thus `JSON.parse` will throw.
`stringify` will just throw earlier.

Reply:
why'd you change it though?

Reply:
typings, `safeStringify` return undefined as of some newer version.